merlin/models/tf/models/base.py

```diff
@@ -1553,7 +1553,7 @@ def predict(
         return out
 
     def batch_predict(
-        self, dataset: merlin.io.Dataset, batch_size: int, **kwargs
+        self, dataset: Union[merlin.io.Dataset, Loader], batch_size: int, **kwargs
```

Comment:
Can you update the docstrings to expose to the user that we are supporting `Loader` as a possible input dataset